content/events/2021-09-16-developer-call-azimuth-scaled.md

```diff
@@ -0,0 +1,24 @@
++++
+title = "Developer Call: Azimuth, Scaled"
+starts = "2021-09-16T09:00:00"
+ends = "2021-09-16T10:00:00"
+timezone = "America/Los_Angeles"
+location = "https://www.youtube.com/watch?v=UScbZ6n7H-4"
+image = "https://secure.meetupstatic.com/photos/event/3/5/f/3/600_498733811.jpeg"
+registration_url = "https://www.meetup.com/urbit-sf/events/280697973/"
+description = "Discussion of Layer 2 Architecture and Implentation"
+dark = true
++++
+
+ETH Gas prices are too high! The growth of Urbit has been greatly hampered this year by the cost of activating planets. Join Philip Monk (~wicdev-wisryt) and the entire L2 team as they present the work completed to date to implement Layer 2 naive rollups. Come see a demo of the new Bridge front end, first migrating a star and spawning some planets, then activating a planet, all on L2.
+
+This call will be a great opportunity for galaxy owners to get their questions answered for the upcoming vote on Layer 2 implementation. It will also be good for anyone in the community that wants a comprehensive look at how naive rollups will work to bring down planet activation costs. If you're technically inclined, be sure to check out [Philip's walkthrough of the naive rollup code](https://www.youtube.com/watch?v=lQdqCwYIBoU) ahead of time so you can ask questions.
+
+Host: Philip Monk, Tlon, CTO (~wicdev-wisryt)
+Guests:
+Jimmy Young, Tlon, Design (~fallyn-balfus)
+Jose Cisneros, Tlon, Infrastructure (~norsyr-torryn)
+Tom Holford, Contributor, UI (~datder-sonnet)
+Mark, Tlon, Infrastructure (~palfun-foslup)
+Poprox, Tlon, Infrastructure (~datnut-pollen)
+Will Galebach, Contributor, UI (~fabnev-hinmur)
```

Comment:
This should all be represented using [[hosts]] and [[guests]] attributes

Reply:
If host and guests have been added above, probably don't need them here any longer.